cmstestsuite/unit_tests/io/rpc_test.py

```diff
@@ -262,7 +262,7 @@ def test_autoreconnect1(self):
         self.sleep()
         self.assertTrue(client.connected)
         self.disconnect_servers()
-        gevent.sleep(0.01)
+        gevent.sleep(0.1)
```

Comment:
Cannot get CI to pass without this change :( Please let me know if this is OK

Reply:
I think this is fine :) The objective is to verify that the client eventually reconnects, and the `sleep()` is there to allow the client to run. See also daf332ff935a7b629dfdf063f7f720a5629202a1.